src/Features/Core/Portable/LanguageServices/SymbolDisplayService/ISymbolDisplayService.cs

```diff
@@ -12,8 +12,21 @@ namespace Microsoft.CodeAnalysis.LanguageService;
 
 internal interface ISymbolDisplayService : ILanguageService
 {
-    Task<string> ToDescriptionStringAsync(SemanticModel semanticModel, int position, ISymbol symbol, SymbolDescriptionOptions options, SymbolDescriptionGroups groups = SymbolDescriptionGroups.All, CancellationToken cancellationToken = default);
-    Task<string> ToDescriptionStringAsync(SemanticModel semanticModel, int position, ImmutableArray<ISymbol> symbols, SymbolDescriptionOptions options, SymbolDescriptionGroups groups = SymbolDescriptionGroups.All, CancellationToken cancellationToken = default);
```

Comment:
no need for these to be on the interface.  they just defer to the other two methods on the type.  moved these to be extensions.